api.go

```diff
@@ -509,19 +516,24 @@ func StopContainer(podID, containerID string) (*Container, error) {
 		return nil, err
 	}
 
+	cImpl, ok := c.(*Container)
```

Comment:
Not needed if we agree that `fetchContainer()` should not return `VCContainer` type.